docs/reference_guide.md

```diff
@@ -3302,6 +3303,31 @@ Attaching 1 probe...
 Operation not permitted
 ```
 
+# 35. `offsetof`: Offset of element in structure
+
+Syntax: `offsetof(struct, element)`
```

Comment:
There is also the form `offsetof(expr, element)` like for `sizeof`, that should be documented here.

Reply:
Fixed, thanks, please review again!!